go/chat/storage/inbox.go

```diff
@@ -985,6 +985,53 @@ func (i *Inbox) SetAppNotificationSettings(ctx context.Context, vers chat1.Inbox
 	return i.writeDiskInbox(ctx, ibox)
 }
 
+// Mark the expunge on the stored inbox
+// Does not delete any messages. Relies on separate server mechanism to delete clear max messages.
+func (i *Inbox) Expunge(ctx context.Context, vers chat1.InboxVers,
```

Comment:
comment here and in avdl that this Expunge on the inbox is only kept up to date for retention. It's ignored for delete-history. We'll probably fix this later.